std/conv.d

```diff
@@ -1857,9 +1857,12 @@ private void testFloatingToIntegral(Floating, Integral)()
         foreach (T; AllNumerics)
         {
             T a = 42;
-            assert(to!string(a) == "42");
-            assert(to!wstring(a) == "42"w);
-            assert(to!dstring(a) == "42"d);
+            string s = to!string(a);
+            assert(s == "42", s);
+            wstring ws = to!wstring(a);
+            assert(ws == "42"w, to!string(ws));
+            dstring ds = to!dstring(a);
+            assert(ds == "42"d, to!string(ds));
```

Comment:
Is this change necessary?

Reply:
I would say yes, while working on std.format this broke and I couldn't see the assert inputs. So this will help the next one person how gets gray hair looking at std.format!double.

Reply:
Thanks for clarifying, I actually missed that you added the inputs as second argument to the asserts.

Reply:
no worries, thanks for taking an interest.

Reply:
Maybe checkaction=context should be enabled for DRuntime (and other repos) unittest builds

Reply:
yes, please